src/__tests__/ShallowWrapper-spec.js

```diff
@@ -83,6 +83,20 @@ describe('shallow', () => {
       expect(wrapper.contains(b)).to.equal(true);
     });
 
+    it('should work with strings', () => {
+      const wrapper = shallow(<div>foo</div>);
+
+      expect(wrapper.contains('foo')).to.equal(true);
+      expect(wrapper.contains('bar')).to.equal(false);
+    });
+
+    it('should work with numbers', () => {
```

Comment:
can you add a counter-test here, that in `<div>1</div>`, `wrapper.contains('1')` is `false`?